cvat/apps/dataset_manager/bindings.py

```diff
@@ -1380,8 +1384,11 @@ def __init__(
         **kwargs
     ):
         dm.SourceExtractor.__init__(
-            self, media_type=dm.Image if dimension == DimensionType.DIM_2D else PointCloud
+            self,
+            subset=instance_meta["subset"],
```

Comment:
`instance_meta` here is reference before assignment, need add `instance_meta = instance_data.meta[instance_data.META_FIELD]` before